### ISSUE TITLE
It is difficult to understand how to upload images and the mode at 300dpi

### ISSUE THREAD

**Issue by new-sankaku** [new-sankaku/stable-diffusion-webui-simple-manga-maker#13]
Repost (post) to issue as another issue

＞Hey man, how do i upload image on website demo? Theres no button for that
ok i got it its just drag and drop but you jmust drop into the empty panels , can you create default canvas with A4 size and at least 300dpi ? current one is square , theres no square manga.
Button to auto allign all the panels when we create them themselves and theyre kinda unevenly placed, it will check distances between each panel /square to match them all to be like 5 pixels from each other or somehtin

**Comment by 3blackbar** [comment]
A4 size in pixels at 300 DPI: 2480 x 3508 pixels
A5 size in pixels at 300 DPI: 1748 x 2480 pixels
Or maybe just keep aspect ratio like in mangas , theyre B6 
B6 size in pixels when using 300 dpi: 2079 x 1476 pixels.

**Comment by new-sankaku** [comment]
I understand, you want to save it in high quality, right?
At the same time, we will also make the Canvas area easier to understand.

Currently, I am scaling up (3x) when saving, so I want to be able to set it arbitrarily. This will increase the number of vertical and horizontal pixels.

**Comment by 3blackbar** [comment]
Yes id say minimum resolution should be 300 DPI so its at least printable and will look good on paper  so 2079 x1476

**Comment by new-sankaku** [comment]
DPI can now be changed based on A4 standards.
However, I think there should be a few to choose from, so I will enhance the functionality.
